net-misc/bird/bird-2.0.8.ebuild

```diff
@@ -12,7 +12,7 @@ LICENSE="GPL-2"
 
 SLOT="0"
 KEYWORDS="~amd64 ~arm64 ~x86 ~x64-macos"
-IUSE="+client debug libssh"
+IUSE="+client debug libssh +lto"
```

Comment:
why are you enabling this by default?

Reply:
Because of https://bugs.gentoo.org/781923#c13

Reply:
won't be merged in this state then, sorry

Reply:
Personally felt the USE wasn't necessary considering this doesn't seem to need a special path/setup for lto to work, users can still add it to their normal FLAGS if they want it (the way want it without the package appending its own override flag at the end).

Reply:
Right, TL:DR: let's not bother with the flag, and disregard upstream here